src/pages/workspaces/workspace/Data.js

```diff
@@ -885,7 +883,11 @@ const WorkspaceData = _.flow(
                       })
                     })
                   ])
-                }, wdsTypes.state)
+                }, wdsTypes.state),
+                h(NoDataPlaceholder, {
+                  buttonText: 'Data Table Status',
+                  onAdd: () => setTroubleshootingWds(true)
```

Comment:
Mini-nit-pick -- but did you want to name `setTroubleshootingWds` to something more generic now/fitting the purpose of the Modal, as it is no longer a "troubleshooter"?